src/Role_Command.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
 /**
- * Manage user roles.
+ * Manages user roles, including creating new roles and resetting to defaults.
+ *
+ * See references for [Roles and Capabilities](https://codex.wordpress.org/Roles_and_Capabilities)and [WP User class](https://codex.wordpress.org/Class_Reference/WP_User).
```

Comment:
Can you add a space before `and`?

Reply:
Will do, then I'll close this PR and submit a new one, correct. Before I do I'll wait to see what folk say about the line-break.

Long and multiple sentences in the descriptions are a problem b/c forced line breaks look OK in the code, for example, of this existing description:
https://github.com/hearvox/shell-command/edit/master/src/Shell_Command.php

But are grammatically wrong, bad for a11y, and look funny in the Handbook:
https://developer.wordpress.org/cli/commands/shell/